react/LocationSearch.tsx

```diff
@@ -56,9 +56,7 @@ const useDebouncedValue = (value: string, delayInMs: number): string => {
   return debouncedValue
 }
 
-const useSuggestions = (
-  searchTerm: string
-): { suggestions: AddressSuggestion[]; loading: boolean } => {
+const useSuggestions = (searchTerm: string): [AddressSuggestion[], boolean] => {
```

Comment:
nice! if you don't want to explicitly type the result, you could've added a `as const` to the return value expression as well

Reply:
Interesting, didn't knew that either, ty.

I'd rather explicitly type this one though, as it increases safety by a little bit, in case someone ever changes the return values.